koji/README.md

```diff
@@ -0,0 +1,20 @@
+---
+title: koji
+homepage: https://github.com/its-danny/koji
+tagline: |
+  🦊 An interactive CLI for creating conventional commits. 
+---
+
+To update or switch versions, run `webi koji@stable` (or `@v2`, `@beta`, etc).
+
```

Comment:
Please add a "Files" section as the first subsection of the Cheat Sheet, as seen here:
- https://webinstall.dev/gpg-pubkey/
- https://webinstall.dev/vim-shfmt/
- https://webinstall.dev/delta/